File: ip-messaging/rest/channels/list-channels/list-channels.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;

class Example {
    static void Main (string[] args) {
        // Find your Account Sid and Auth Token at twilio.com/user/account
        const string accountSid = "accountSid";
        const string authToken = "authToken";
        const string serviceSid = "serviceSid";

        //Retrieve the list of channels
        var client = new TwilioIpMessagingClient(accountSid, authToken);
        ChannelResult channels = client.ListChannels(serviceSid);
        Console.WriteLine(channels);
    }
}

File: ip-messaging/rest/channels/list-channels/list-channels.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest.ip_messaging import TwilioIpMessagingClient

# Initialize the Client
account = "ACCOUNT_SID"
token = "AUTH_TOKEN"
client = TwilioIpMessagingClient(account, token) 

# List the channels
service = client.services.get(sid="SERVICE_SID")
for c in service.channels.list():
    print c.sid